src/backend/profile/profile.service.ts

```diff
@@ -10,9 +10,10 @@ export async function getPublishedProfilesPayload() {
     },
     include: includeObject.include,
   })
-  const serializedProfile = publishedProfiles.map(
-    serializeProfileToProfileModel,
-  )
+
+  const shuffledProfiles = publishedProfiles.sort(() => Math.random() - 0.5)
```

Comment:
**nit:** We should do this shuffle in ProfilesList component instead of service logic